File: MPI/mpicheck/tests/functions/2.0/MPI_Info_get_valuelen.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       LOGICAL var_3
       INTEGER var_4
        call mpi_info_get_valuelen(var_0, var_1, var_2, var_3, var_4)
       call pmpi_info_get_valuelen(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Info_get_valuelen.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Info var_0;
    const char *var_1;
    int *var_2;
    int *var_3;
    int ret;
    /* calls */
    ret = MPI_Info_get_valuelen(var_0, var_1, var_2, var_3);
    ret = PMPI_Info_get_valuelen(var_0, var_1, var_2, var_3);
    return 0;
}
